src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -175,6 +187,81 @@ func (c *controller) WaitAndUpdateNodesStatus(status *ControllerStatus) {
 	c.log.Infof("Done waiting for all the nodes. Nodes in error status: %d\n", hostsInError)
 }
 
+func (c *controller) hackDNSAddressConflict(wg *sync.WaitGroup) {
+
+	c.log.Infof("Making sure service %s can reserve the .10 address", dnsServiceName)
+
+	defer wg.Done()
+	networks, err := c.kc.GetServiceNetworks()
```

Comment:
don't we have service network in cluster object?

Reply:
Can you point me to it? I couldn't find a better way to read this, used `networkType` https://github.com/openshift/assisted-installer/blob/371646af01255d3ae798920e9974f1fdd0890e46/src/k8s_client/k8s_client.go#L233 as an example

Reply:
we have for sure but maybe better to take from ocp cluster and not from service in anycase